src/components/question-editor/QuestionEditor.tsx

```diff
@@ -130,63 +156,110 @@ const QuestionEditor = () => {
     </Box>
   ))
 
+  const disabledButton =
+    type === 'openAnswer'
+      ? !!text && !!openAnswer
```

Comment:
i think there is no need to change types to boolean cause if there will be empty string in `text` and condition will be false

Reply:
You are right. I fixed it.